### PR TITLE
Began adding configuration for metric beat

### PR DESCRIPTION
**Pull request by blarghmatey**
#### What's this PR do?
Adds configuration for installing and setting up metricbeat for our servers

### CODE REVIEW

File: pillar/elastic_stack/beats/init.sls [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+              - cpu
+              - filesystem
+              - load
+              - memory
+              - network
+              - process
+              - process_summary
+              - uptime
+            enabled: 'true'
+            period: 1s
```
Comment (shaidar) [inline pillar/elastic_stack/beats/init.sls]: Isn't this too low? Would for example `5s` or `10s` still give us a good picture?

**Comment by shaidar** [comment]
We probably need to evaluate the need for increased disk space on the ES logging cluster once we start deploying this

File: pillar/elastic_stack/beats/init.sls (hunk 1 above)
Comment (blarghmatey) [reply pillar/elastic_stack/beats/init.sls]: In order for it to be actually useful it's best to go as low as possible. If we can't handle the volume of writes at 1s granularity then we can try going coarser, but this is a good place to start.

**Comment by blarghmatey** [comment]
I'm sure we'll need to scale up the cluster regardless as we add more logs and do more work on adding visibility to our full infrastructure. We will also be wanting to add packetbeat and auditbeat to the majority of our instanaces, and in some places we'll want to run heartbeat.

**Comment by shaidar** [comment]
👍

**Pull request merged**